src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -173,6 +173,7 @@ def load_arguments(self, _):
         c.argument('service_cidr')
         c.argument('vnet_subnet_id')
         c.argument('workspace_resource_id')
+        c.argument('skip-subnet-role-assignment', required=False, action='store_true')
```

Comment:
remove the `required=False`, this will be inferred from the function signature.

Reply:
Ok, don't know this. will make a change